index.html

```diff
@@ -8,33 +8,267 @@
     <title>index</title>
     <link rel="stylesheet" type="text/css" href="styles.css">
     <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/c4ef94e39c.js"></script>
+
+
+
 </head>
 
 <body>
     <div class="main-image-container">
         <div class="navbar">
```

Comment:
you could use a `<nav>` element here instead, more semantic

Reply:
done